### PR TITLE
CHANGELOG: Format Markdown

### PR DESCRIPTION
**Pull request by olleolleolle**
- Fix typos
- Link to PRs
- Use reference-style links to repeating links

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -48,25 +48,25 @@
 
 * Incorporate newer libpg_query updates in 10-1.0.3 and 10-1.0.4
   * Adds support for running on ARM
-  * Fixes asprintf warning during builds
+  * Fixes a sprintf warning during builds
```
Comment (lfittl) [inline CHANGELOG.md:51]: This is actually intended to reference `asprintf` (the C method)

**Review by lfittl — COMMENTED** [review]
One minor comment, otherwise this looks great!

File: CHANGELOG.md (hunk 1 above)
Comment (olleolleolle) [reply CHANGELOG.md]: Thanks, fixed!

**Comment by olleolleolle** [comment]
@lfittl There!

**Pull request merged**